source/jormungandr/jormungandr/parking_space_availability/bss/tests/bss_provider_manager_test.py

```diff
@@ -85,6 +85,28 @@ def realtime_place_handle_test():
     places_with_stands = manager.handle_places(places)
     assert 'stands' in places_with_stands[0]['poi']
 
+def realtime_place_handle_test():
+    """
+    test correct handle include bss stands
+    """
+    places = {
+        'embedded_type': 'poi',
+        'distance': '0',
+        'name': 'Cit\u00e9 Universitaire (Laval)',
+        'poi': {
+            'poi_type': {
+                'name': 'station vls',
+                'id': 'poi_type:amenity:bicycle_rental'
+            }
+        },
+        'quality': 0,
+        'id': 'poi:n3762373698'
+    }
+    app.config['BSS_PROVIDER'] = CONFIG
+    manager = BssProviderManager()
+    place_with_stands = manager.handle_place(place)
+    assert 'stands' in place_with_stands['poi']
```

Comment:
I know it's the same as the test above, but it would be nice to check the validity of the stands object in itself.

you can check that the values inside are the one given by the mock